base_synchro/models/base_synchro_obj.py

```diff
@@ -45,15 +44,16 @@ class BaseSynchroObj(models.Model):
     avoid_ids = fields.One2many('base.synchro.obj.avoid', 'obj_id',
                                 'Fields Not Sync.')
 
-    @api.model
-    def get_ids(self, obj, dt, domain=None, action=None):
-        """Method to get ids."""
-        if action is None:
-            action = {}
-        return self._get_ids(obj, dt, domain, action=action)
+#    @api.model
+#    def get_ids(self, obj, dt, domain=None, action=None):
+#        """Method to get ids."""
+#        if action is None:
+#            action = {}
+#        return self._get_ids(obj, dt, domain, action=action)
 
+#    def _get_ids(self, obj, dt, domain=None, action=None):
     @api.model
-    def _get_ids(self, obj, dt, domain=None, action=None):
+    def get_ids(self, obj, dt, domain=None, action=None):
```

Comment:
why removing? We hav both the methods being used. isnt it?

Reply:
_get_ids method is called from  synchronize method. And get_ids is called from _get_ids. So I have removed _get_ids method and directly called get_ids method.